.github/workflows/action-registry-index-update-release.yml

```diff
@@ -0,0 +1,14 @@
+name: Creating the PR to update the version of buildpacks/github-actions on buildpacks/registry-index
+on:
+  release:
+    types: [published]
+jobs:
+  myEvent:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Repository Dispatch
+        uses: peter-evans/repository-dispatch@v1
```

Comment:
```suggestion
        uses: peter-evans/repository-dispatch@v1.1.13
```

Reply:
Thanks for the review and suggestion @jkutner! Fixed it :)